app/controllers/applications_controller.rb

```diff
@@ -0,0 +1,55 @@
+class ApplicationsController < ApplicationController
+
+  def create
+    create_filter
+  end
+
+  def update
+    update_filter
+  end
+
+  def show
+    show_filter
+  end
+
+  private
+  def application_params
+    params.permit(
+      :name,
+      :street_address,
+      :city,
+      :state,
+      :zipcode,
+      :description,
+      :status)
+  end
+
+  def create_filter
```

Comment:
The content of these controller actions look good! I'd say none of these are so complex that they *need* a helper filter method, so it would be okay to just have them living in the CRUD actions above.